src/screens/shared/farmProfileEdit/FarmProfileEditGapStatus.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { withStyles } from '@material-ui/core/styles';
+import EditGapStatusDropdown from './components/EditGapStatusDropdown';
+
+const styles = {
+  row: {
+    display: 'flex',
+    justifyContent: 'space-around'
+  },
+  column: {
+    display: 'flex',
+    flexDirection: 'column',
+    margin: '0px 24px',
+    width: '100%'
+  }
+};
+
+class FarmProfileEditGapStatus extends React.PureComponent {
+  onChange = prop => event => {
+    const { values, handleChange } = this.props;
+
+    const gapCertificationValues = {
+      ...values,
+      [prop]: event.target.value
+    };
+    handleChange(gapCertificationValues);
+  };
+
+  mapPropsToLabels = () => {
+    return {
+      referred: 'Farm Referred',
+      applied: 'Farm Applied',
+      accepted: 'Farm Accepted',
+      safetyPlan: 'Food Safety Plan Complete',
+      riskAssessment: 'Risk Assessment',
+      mockRecall: 'Mock Recall Complete',
+      internalAudit1: 'Internal Audit Complete (1)',
+      internalAudit2: 'Internal Audit Complete (2)',
```

Comment:
`mapPropsToLabels` assumes it knows the names of state variables representing the GAP certificaiton status and maps them to text labels. 

@phoebeli23 Is the Airtable open-access for NSEVP? I'd like to change the schema to camel-case for an easier time updating edit form data in the future.